packages/ui/atomic/create-atomic-component/tests/test.spec.ts

```diff
@@ -49,12 +53,6 @@ describe(PACKAGE_NAME, () => {
       testDirname: 'valid-arg',
       packageName: 'valid-component-name',
     },
-    {
-      describeName: 'when called with a component name without hyphen',
-      args: ['nohypen'],
-      testDirname: 'invalid-arg',
-      packageName: 'atomic-nohypen',
-    },
```

Comment:
No need to test this use case because it will never pass the validation rules. The cli will simply throw an error message. this is tested in utils/create-atomic-commons/src/validation.spec.ts